in_toto/in_toto_keygen.py

```diff
@@ -99,11 +104,17 @@ def main():
 
   try:
     if args.prompt:
-      in_toto.util.prompt_generate_and_write_rsa_keypair(args.name, args.bits)
+      if args.type == in_toto.util.KeyType.RSA:
+        in_toto.util.prompt_generate_and_write_rsa_keypair(args.name,
+                                                           args.bits)
+      elif args.type == in_toto.util.KeyType.ED25519:    # pragma: no branch
```

Comment:
Using `# pragma: no branch` worked as we know for a fact that this is essentially an else statement.

Reply:
I don't quite understand why this line isn't being covered.  Shouldn't the unit test for `arg.prompt=True` and `args.type=ed25519` cover it?

Reply:
The coverage error was for this `110 -> 112`. Basically it was unable to find any case where both of the ifs were skipped (its not possible).

Reply:
Ahh, all this time I've been thinking it was line coverage that was missing - it's branch coverage.

What I normally do is explain in a comment why the # pragma is being added.  It's often beneficial to document any assumptions made by the code.  In this case, you want to state that argparse enforces the values specified in --keytype.  However, will the if-elif construction still work if the `choices` parameter of add_argument() is later updated?

You might also want to use an actual else case.  For instance:

```Python


if arg.keytype == 'ed25519':
   ...
elif arg.keytype == 'rsa':
   ...
else: # pragma: no branch
    raise Error('Unsupported keytype: ' + str(arg.keytype))
```

Or raise the exception up front, as [we recommend in our code style guidelines](https://github.com/secure-systems-lab/code-style-guidelines#nesting).